buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -118,6 +117,7 @@ object Deps {
         private const val version_ktx = "1.1.0-alpha02"
         const val ANDROIDX = "androidx.security:security-crypto:$version"
         const val KTX = "androidx.security:security-crypto-ktx:$version_ktx"
+        const val ROOTBEER = "com.scottyab:rootbeer-lib:0.0.8"
```

Comment:
it seems not to be used

Reply:
I can remove it, but it is generally used for all security projects in my practice.